docs/helm-values-documenation.md

```diff
@@ -11,7 +11,7 @@
 | `appgw.resourceGroup` | Default is agent node pool's resource group derived from CloudProvider config | Name of the Azure Resource Group in which App Gateway was created. Example: `app-gw-resource-group` |
 | `appgw.name` | | Name of the Application Gateway. Example: `applicationgatewayd0f0` |
 | `appgw.environment`| `AZUREPUBLICCLOUD` | Specify which cloud environment. Possbile values: `AZURECHINACLOUD`, `AZUREGERMANCLOUD`, `AZUREPUBLICCLOUD`, `AZUREUSGOVERNMENTCLOUD` |
-| `appgw.shared` | false | This boolean flag should be defaulted to `false`. Set to `true` should you need a [Shared App Gateway](setup/install-existing.md#multi-cluster--shared-app-gateway). |
+| `appgw.shared` | false | This boolean flag should be defaulted to `false`. Set to `true` should you need a [Shared App Gateway](how-tos/prevent-agic-from-overwriting.md). |
```

Comment:
nitpick not super related to the PR but this filename `docs/helm-values-documenation.md` has a typo

Reply:
will make note a fix in a subsequent pr.